### DISCUSSION TITLE
<title>how to add the Google consule html code in GitHub to verification my ownership

### DISCUSSION THREAD

**Discussion by muthukumarkarunakaran** [community/community#48229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

how to add the Google consule html code in GitHub to verification my ownership

**Comment by muthukumarkarunakaran** [comment]
People kindly give me the solution please humble request

**Comment by cehfisher** [comment]
Hi @muthukumarkarunakaran - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
